Status/index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+define("ROOTDIR", "../");
+define("REAL_ROOTDIR", "../");
+
+require_once REAL_ROOTDIR."src/initializer.php";
+use \Catalyst\Page\Values;
+use \Catalyst\User\User;
+
+define("PAGE_KEYWORD", Values::STATUS[0]);
+define("PAGE_TITLE", Values::createTitle(Values::STATUS[1], []));
+define("NO_HEADER", 1);
+
+if (User::isLoggedIn()) {
+	define("PAGE_COLOR", $_SESSION["user"]->getColor());
+} else {
+	define("PAGE_COLOR", Values::DEFAULT_COLOR);
+}
+
+require_once Values::HEAD_INC;
+?></div>
+	<?php include('status.html'); ?>
```

Comment:
It's a bit more standard to use the require_once syntax like the others. For this it could be:
require_once REAL_ROOTDIR."Status/status.html";

However, it would be even better to physically insert the HTML here directly since it's only being used in one place.